docs/api-overview.mdx

```diff
@@ -398,18 +399,17 @@ This endpoint returns a list of credential requests that the partner has defined
 
 #### Response Body:
 
-```typescript title="CredentialRequest"
+```typescript title="CredentialRequestsResponse"
 {
-  "credentialRequests": {
-    "type": string, // credential type
-    "issuers": string[], // list of acceptable brandIds; if empty, all issuer brands are valid
-    "required": boolean, // whether the credential is required
-  }[]
+  "credentialRequests": CredentialRequest[]
 }
 ```
 
+Type reference for:
+- [CredentialRequest](https://docs.verified.inc/types/interfaces/credentialrequest.html)
```

Comment:
I think this is generated from the old types repo that we used in our old architecture. I'm not sure if we have anything like it for the current architecture

Reply:
Should then I add the same block piece as in Check User Credentials? Didn't saw a api references either.

Reply:
Yeah that's fine